Fix: #22007 - Allow for ImgButton to be updated. (#22012)
OpenRCT2/OpenRCT2: distribution/changelog.txt

- Change: [#7248] Small mini-maps are now centred in the map window.
- Fix: [#13294] Map corners are cut off in some directions (original bug).
- Fix: [#21974] No reason specified when attempting to place benches, lamps, or bins on path with no unconnected edges (original bug).

> - Fix: [#22012] [Plugin] Allow for image updates for the ImgButton widget.

Reviewer: For fixes, always list the problem, rather than the solution (even if that means deviating from the title of the bug report).

```suggestion
- Fix: [#22012] [Plugin] Images on ImgButton widgets cannot be updated.
```
